doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -388,6 +388,17 @@ Libraries for networking
 
   * Updated to request proprietary PSM mode for ``SOC_NRF9151_LACA`` and ``SOC_NRF9131_LACA`` in addition to ``SOC_NRF9161_LACA``.
 
+* :ref:`lib_lwm2m_client_utils` library:
+
+  * The following initialization functions have been deprecated:
```

Comment:
We should have a comment somewhere saying what to do instead. You could mention it here. And I'm not sure if you also need a statement in the migration guide.

Reply:
Good point. I modified the text to `following initialization functions have been deprecated as these modules are now initialized automatically on boot`

And 
> :c:func:`lwm2m_init_firmware` is deprecated in favour of :c:func:`lwm2m_init_firmware_cb`

So basically there is nothing to do other than remove those obsolete _init() functions.